afl-fuzz.c

```diff
@@ -487,8 +490,20 @@ static void bind_to_free_cpu(void) {
   }
 
   closedir(d);
+  if (cpu_to_bind_given) {
+
+    if (0 <= cpu_to_bind && cpu_to_bind < cpu_core_count) {
```

Comment:
`0 <= cpu_to_bind && ` can be removed since `cpu_to_bind` is unsigned

Reply:
also, let's decrease the nested indentation by rewriting this in the following way:

```
    if (cpu_to_bind >= cpu_core_count)
      FATAL("The CPU core id to bind should be between 0 and %u", cpu_core_count - 1);

    if (cpu_used[cpu_to_bind])
      FATAL("The CPU core #%u to bind is not free!", cpu_to_bind);

    i = cpu_to_bind;
```

Reply:
Done, thanks!